debian/README.etc

```diff
@@ -0,0 +1,17 @@
+Configuring the Kolibri Debian package:
+
+Kolibri is normally configured in its own environment, for instance in
+its main configuration file `~/.kolibri/options.ini` or through command
+line options such as `kolibri start --port=1234`.
```

Comment:
> command line options

Command line options are not currently a supported (publicly documented) way of configuring Kolibri

Reply:
This does not count? :confused: 

https://kolibri.readthedocs.io/en/latest/advanced.html#working-with-kolibri-from-the-command-line

Reply:
Nope: configuration options are drawn first from environment variables, and second from `options.ini`, but never from the command line.

https://github.com/learningequality/kolibri/blob/develop/kolibri/utils/options.py

Reply:
Gotcha, I guess my understanding of _configuration_ was too broad! :+1:

Reply:
Oh gonna fix that - thanks for the catch @indirectlylit